Sources/Extensions/Foundation/FileManagerExtensions.swift

```diff
@@ -56,5 +56,17 @@ public extension FileManager {
         return nil
     }
 
+    /// SwifterSwift: Save Encodable object to a given URL using PropertyListEncoder.
+    ///
+    /// - Parameters:
+    ///   - object: Object of type T that conforms to Encodable protocol.
+    ///   - encoder: PropertyListEncoder.
+    ///   - url: URL to save object to.
+    ///   - attributes: A dictionary containing the attributes to associate with the new file. You can use these attributes to set the owner and group numbers, file permissions, and modification date. For a list of keys, see FileAttributeKey. Set to nil by default.
+    /// - Throws: Throws any errors thrown by PropertyListEncoder or FileManager.
+    public func encodePlist<T: Encodable>(_ object: T, using encoder:PropertyListEncoder=PropertyListEncoder(), to url: URL, attributes: [FileAttributeKey:Any]? = nil) throws {
```

Comment:
Look out for SwiftLint issues.
```suggestion
    public func encodePlist<T: Encodable>(_ object: T, using encoder: PropertyListEncoder = PropertyListEncoder(), to url: URL, attributes: [FileAttributeKey:Any]? = nil) throws {
```